src/openMVG/calibration/bestImages.cpp

```diff
@@ -133,8 +132,12 @@ void selectBestImages(const std::vector<std::vector<cv::Point2f> >& imagePoints,
   }
   else
   {
-    OPENMVG_LOG_DEBUG("Info: Less valid frames (" << validFrames.size() << ") than specified maxCalibFrames (" << maxCalibFrames << ").");
-    bestImagesIndexes = validFrames;
+    OPENMVG_LOG_DEBUG("Info: Less valid frames (" << imagePoints.size() << ") than specified maxCalibFrames (" << maxCalibFrames << ").");
+    bestImagesIndexes.resize(imagePoints.size());
+    for(std::size_t i = 0; imagePoints.size(); ++i)
```

Comment:
it's strange as there are lots of other places where it is used
https://github.com/alicevision/openMVG/search?utf8=%E2%9C%93&q=iota